snmp/metadata.csv

```diff
@@ -1463,6 +1463,30 @@ snmp.sentry3.towerLineFrequency,gauge,,,,"[SENTRY3-MIB] The frequency of the inp
 snmp.sentry3.towerPowerFactor,gauge,,,,"[SENTRY3-MIB] The overall power factor of the tower circuit. A non- negative value indicates the overall power factor in hundredths. A negative value indicates that the overall power factor was not able to be measured.",0,snmp,,
 snmp.sentry3.towerVACapacity,gauge,,,,"[SENTRY3-MIB] The total apparent power capacity of the tower circuit. A non-negative value indicates the maximum total apparent power in Volt-Amps. A negative value indicates that the total apparent power capacity was not available.",0,snmp,,
 snmp.sentry3.towerVACapacityUsed,gauge,,,,"[SENTRY3-MIB] The used percentage of the tower circuit total apparent power capacity (towerApparentPower / towerVACapacity x 100). A non-negative value indicates the percentage of capacity used in tenths. A negative value indicates that the percentage of capacity used was not available.",0,snmp,,
+snmp.sentry4.BranchCurrent,gauge,,,,"[SENTRY4-MIB] The measured current on the branch in hundredth Amps.",0,snmp,,
```

Comment:
I'm wondering if we should prefix servertech metrics with `servertech`.

Currently we [have metrics like](https://github.com/DataDog/integrations-core/blob/805e2aa71b2b7865649bb103e41d2d8cb76d6658/snmp/metadata.csv#L1443-L1489)
- snmp.sentry3.*
- snmp.sentry4.*

But I think it might be better to put everything under servertech:
- snmp.servertech.sentry3.*
- snmp.servertech.sentry4.*

This can help discoverability when user types snmp.servertech... with autocomplete.